packages/zipkin-instrumentation-request-promise/src/request.js

```diff
@@ -0,0 +1,106 @@
+import request from 'request-promise';
+import _ from 'lodash';
+import {Instrumentation} from 'zipkin';
+import {Deferred} from './promise';
+
+const normalizeParameter = function(param) {
+  let options = param;
+  if (!param) {
+    options = {};
+  } else if (typeof param === 'string') {
+    options = {
+      uri: param,
+    };
+  }
+  return options;
+};
+
+export default class Request {
+  constructor(tracer, serviceName, remoteServiceName) {
```

Comment:
Sorry I only meant remove "serviceName" not "remoteServiceName" the latter can't be a tracer-default as it is endpoint specific ex https://github.com/openzipkin/zipkin-js/blob/ee844c9249b47872444bdc5c4514f39df27a9714/packages/zipkin-instrumentation-cujojs-rest/README.md

Reply:
if someone can remove the serviceName parameter it would be good. "serviceName" needs to be consistent across a trace and having it here begs people to do it inconsistently. This can be done in a separate commit, but should be done before release